BitcoinChangeRates/BitcoinChangeRates/ChangeRates/ChangeRatesViewController.swift

```diff
@@ -27,6 +27,7 @@ final class ChangeRatesViewController: UIViewController, UITableViewDataSource {
         setupLayout()
         setupLoadingView()
         fetchData()
+        
```

Comment:
Please say hello to me, then i close the conversation !

Reply:
Helllo